docs/en/03_Upgrading/index.md

```diff
@@ -10,9 +10,66 @@ your SilverStripe installation means overwriting files, flushing the cache and u
 See our [upgrade notes and changelogs](/changelogs/4.0.0) for 4.0.0 specific information, bugfixes and API changes.
 </div>
 
-## Composer
+## Understanding and planning your upgrade {#planning}
```

Comment:
The detailed changelogs (4.0.0, 4.1.0 and 4.2.0) should all point back to this upgrading guide with clear wording. Something like:

> This changelog describes API changes in detail. Use our [step-by-step guide for upgrading to SilverStripe 4](...) to perform an upgrade on your module or codebase.

Reply:
Done.